tests_new/integration_tests/conftest.py

```diff
@@ -0,0 +1,133 @@
+import os
+from dataclasses import dataclass
+
+import pytest
+from dataall.base.loader import load_modules, ImportMode, list_loaded_modules
+from glob import glob
+
+from tests_new.integration_tests.client import Client
+
+load_modules(modes=ImportMode.all())
+
+
+collect_ignore_glob = []
+
+
+def ignore_module_tests_if_not_active():  # TODO: maybe move to commons between tests
```

Comment:
Isn't there a way to query which modules are active from the API directly?
Don't we get different response when the query is missing vs when something goes wrong?

Reply:
If a module is not enabled the API call won't exist, so the response will be something like (I just ran a query with a typo): `"Cannot query field 'listTenantPermission' on type 'Query'. Did you mean 'listTenantPermissions' or 'listTenantGroups'?"`

It is a bit hacky, I have been looking for ways to define the test scope of an API call using graphql, with no luck. Another option is to list the available Queries and Mutations (no with API GW but maybe with AppSync introspection?). But we still need to map them to modules or we run tests based on each API call....

Reply:
I don't think it's hacky, you can create a decorator and check which APIs must be available, otherwise test will be disabled. Something like `disable_if(api='Query.listTenantGroup')`

Another option is to provide a real query that returns all the enabled modules and enable it only on dev stages 

Also introspection is not a bad idea, you can [enable it in Apollo server as well](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#introspection), is not an AppSync specific feature.